grafana-plugin/e2e-tests/integrations/maintenanceMode.test.ts

```diff
@@ -40,7 +40,7 @@ test.describe('maintenance mode works', () => {
   const enableMaintenanceMode = async (page: Page, mode: MaintenanceModeType): Promise<void> => {
     await _openIntegrationSettingsPopup(page, true);
     // open the maintenance mode settings drawer + fill in the maintenance details
-    await page.getByTestId('integration-start-maintenance').click();
+    await page.getByText('Start Maintenance').click();
```

Comment:
hmm, why query by text instead of unique id?

Reply:
test id has been removed as part of my changes and I decided to not recreate it because selecting by text is also a good practice in e2e tests world